src/Company/Recommendation_v1.0.py

```diff
@@ -8,36 +8,37 @@ class RecommendationRequest(CamelCaseModel):
     keywords: str = Field(
         ...,
         title="Keywords",
-        description="Keyword data to base recommendations on",
+        description="Keyword data to base recommendations on.",
         examples=["Looking for data product companies to invest on"],
     )
 
 
 class Recommendation(CamelCaseModel):
     score: int = Field(
-        ..., description="Recommendation score of the company", examples=[231]
+        ..., description="Recommendation score of the company.", examples=[231]
     )
     company_id: str = Field(
         ..., title="Company ID", description="Company ID", examples=["2464491-9"]
```

Comment:
Should this description be changed to a proper description and end with a dot also?

A small hint, that you can force black to put each of these keyword arguments on their own line by adding a comma after the last one, in case you prefer that for readability.